src/mixs/schema/radiocarbon-dating.yml

```
@@ -114,12 +114,12 @@ slots:
     multivalued: false
     required: true
     recommended: true
-  c14_date_lab_code:
+  c14_age_lab_ID:
```

```suggestion
  c14_age_lab_id:
```

Must be all lower case